src/tests/ftest/util/command_utils.py

```diff
@@ -25,6 +25,7 @@
 import re
 import time
 import signal
+import subprocess
```

Comment:
Seeing as we only need the `TimeoutExpired` exception, I believe this would be more appropriate:
```suggestion
from subprocess import TimeoutExpired
```